### PR TITLE
fix: Deprecated commands in actions set-output

### PR DESCRIPTION
**Pull request by Lisa-Stubert**
#174 @ff6347

### CODE REVIEW

**Comment by ff6347** [comment]
@Lisa-Stubert Wait if #192 passes. if so merge it first. After that make a small commit to your branch and see if the build action runs and passes. in the build action you should see if your PR uses the right node version

**Comment by ff6347** [comment]
@Lisa-Stubert Test is failing :/

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -10,15 +10,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, 'skip ci')"
-
-    steps:
-      - name: Read .nvmrc
-        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
-        id: nvm
-      - uses: actions/checkout@v3
```
Comment (ff6347) [inline .github/workflows/build.yml:18]: you removed the checkout step. Without this step the source code of the repo is not available

Comment (ff6347) [reply .github/workflows/build.yml:18]: you need `      - uses: actions/checkout@v3`

Comment (ff6347) [reply .github/workflows/build.yml:18]: 👍🏾

**Comment by Lisa-Stubert** [comment]
thank u @ff6347

**Pull request merged**